packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/actions.ts

```diff
@@ -11,10 +11,11 @@ import { TIERS } from '../../modules/profile/model'
 export const verifyIdentity = (
   tier = TIERS[2],
   needMoreInfo = false,
-  origin: ModalOriginType
+  origin: ModalOriginType,
+  metadata?: any
```

Comment:
create types for each metadata

`origin: SwapMetadataType | InterestMetdataType`

Reply:
add basic support for `callback`